src/WasmDis.ts

```diff
@@ -1056,6 +1078,7 @@ export class WasmDisassembler {
           this._indentLevel = 0;
           this._labelIndex = 0;
           this._backrefLabels = this._labelMode === LabelMode.Depth ? null : [];
+          this.logStartOfBodyFunctionOffset();
```

Comment:
It's unclear to me whether we are logging the byte offset corresponding to the start of the function header here or the end (start of the code). I think we want the latter for our purposes, but the test seems to imply it's the former.

Reply:
we log the position of start of function header and right after the function end. I'm not sure what you propose we should change here?

Reply:
So then in the first CODE_OPERATOR after the BEGIN_FUNCTION_BODY, we would have the position of the first breakable location within the function, which is what we are looking for? I would prefer that we get that offset, or maybe both in an object.

Reply:
Agree, any benefit of getting BEGIN_FUNCTION_BODY?

Reply:
No clear need for it at this time.

Reply:
thanks, it is done :)